autogen.sh

```diff
@@ -38,6 +38,8 @@ aclocal \
 && automake --add-missing \
 && autoconf
 
+(cd test/pjdfstest && autoreconf -ifs && ./configure)
```

Comment:
Any ideas on what is the correct way to do this?

Reply:
Setting pjdfstest as a submodule is a bit thoughtful.

I want the s3fs repository to be able to work independently.
I don't think I agree with mixing the pjdfstest build with the s3fs build.
The pjdfstest build is the `environment` required for the s3fs build(and test), and I think it's better to separate this part.

If you want to use pjdfstest in s3fs test, what about the following method?
- In github actions, git clone pjdfstest under the test directory(this can be done with ci.yml. You can prepare a helper script if necessary)
- And in the pjdfstest directory, do that build

How about performing this step only during CI?

Anyone can deploy it on localhost other than CI by preparing a dedicated helper script.
(Alternatively, you may prepare another repository(ex. s3fs-fuse/s3fs-pjdfstest, etc.) and dedicate it. But this is still an idea I just come up with, so it may be wrong.)

Reply:
I came up with another one.
You may want to call the script as a PHONY target in the `Makefile (.am)` of the test directory.
(Clone and build pjdfstest in the script)

Reply:
I made a example. (see proto_pjdfstest branch in my personal repo)
https://github.com/ggtakec/s3fs-fuse/compare/master...proto_pjdfstest

I haven't implemented a "test" using pjdfstest yet...(because I don't know how you want to do it)
This way, pjdfstest can be made independent for now, which is in line with my wishes.

Reply:
I reworked this to download the tarball which is cleaner.